src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1399,25 +1399,43 @@ bool NetworkOPsImp::checkLastClosedLedger (
 }
 
 void NetworkOPsImp::switchLastClosedLedger (
-    Ledger::pointer newLedger, bool duringConsensus)
+    Ledger::pointer newLCL, bool duringConsensus)
```

Comment:
This is code that is critical to get right, and modeled after the code in `LedgerConsensusImp`. I used the same names so that anyone who tries to understand the code will see names they can understand. `openLedger` is a bool in my *openmeta* branch so I changed it here to prevent a conflict later.